daisy_integration_tests/rhel_8_translate.wf.json

```diff
@@ -0,0 +1,99 @@
+{
+  "Name": "rhel-8-licensed-translate-test",
+  "Sources": {
+    "post_translate_test.sh": "./scripts/post_translate_test.sh"
+  },
+  "Vars": {
+    "image_name": {
+      "Value": "rhel-8-licensed-translate-test-${ID}"
+    },
+    "source_image": {
+      "Value": "projects/compute-image-tools-test/global/images/rhel-8-import"
+    },
+    "test-id": {
+      "Value": "",
+      "Description": "The ID of this test run."
+    }
+  },
+  "Steps": {
+    "create-disk-from-image": {
+      "CreateDisks": [
+        {
+          "name": "translate-me",
+          "sourceImage": "${source_image}"
+        }
+      ]
+    },
+    "create-test-disk": {
+      "CreateDisks": [
+        {
+          "name": "disk-import-test",
+          "sourceImage": "${image_name}",
+          "type": "pd-ssd"
+        }
+      ]
+    },
+    "create-test-instance": {
+      "CreateInstances": [
+        {
+          "disks": [
+            {
+              "source": "disk-import-test"
+            }
+          ],
+          "machineType": "n1-standard-4",
+          "name": "inst-import-test",
+          "StartupScript": "post_translate_test.sh"
+        }
+      ]
+    },
+    "delete-image": {
+      "DeleteResources": {
+        "Images": [
+          "${image_name}"
+        ]
+      }
+    },
+    "translate-disk": {
+      "Timeout": "90m",
+      "IncludeWorkflow": {
+        "Path": "../daisy_workflows/image_import/enterprise_linux/translate_rhel_8_licensed.wf.json",
+        "Vars": {
+          "image_name": "${image_name}",
+          "source_disk": "translate-me"
+        }
+      }
+    },
+    "wait-for-test-instance": {
+      "Timeout": "30m",
+      "WaitForInstancesSignal": [
+        {
+          "Name": "inst-import-test",
+          "SerialOutput": {
+            "Port": 1,
+            "SuccessMatch": "PASSED:",
+            "FailureMatch": "FAILED:",
+            "StatusMatch": "STATUS:"
+          }
+        }
+      ]
+    }
+  },
+  "Dependencies": {
```

Comment:
Sort these in chronological order for easier understanding of the full workflow

Reply:
sgtm